cabal.project

```diff
@@ -3,7 +3,8 @@ packages:
   servant-client/
   servant-client-core/
   servant-http-streams/
-  servant-docs/
+-- Tests failing with Cabal (TODO: investigate)
+--  servant-docs/
```

Comment:
FWIW I was able to get tests to pass by testing twice, with each field.
https://github.com/haskell-servant/servant/pull/1401/files#diff-1d3cfc9328ae9300f3c558d76c2bab79a5eee646734eac7500f828cd318275f1

Reply:
That's good to know ! There is a pending PR ( #1397 ) that fixes this problem in a slightly different way. We'll ponder what the best approach is.